packages/api/src/tx/regen-message-type-registry.ts

```diff
@@ -25,7 +25,7 @@ import {
 } from '../generated/regen/ecocredit/marketplace/v1/tx';
 import { GeneratedType } from '@cosmjs/proto-signing';
 
-const ecocreditRegistry: Array<[string, GeneratedType]> = [
+const ecocreditRegistry: ReadonlyArray<[string, GeneratedType]> = [
```

Comment:
`ReadonlyArray` is more in line with the established convention for type registries.